digitalocean/spaces/spaces_entities.go

```diff
@@ -45,6 +46,50 @@ func spacesBucketSchema() map[string]*schema.Schema {
 	}
 }
 
+func spacesKeySchema() map[string]*schema.Schema {
+	return map[string]*schema.Schema{
+		"name": {
+			Type:        schema.TypeString,
+			Required:    true,
+			Description: "A name for the key. This is used to identify the key in the DigitalOcean control panel.",
+		},
+		"grant": {
+			Type:        schema.TypeList,
+			Optional:    true,
+			Description: "A list of grants to apply to the key. Can be left empty to apply no grants.",
+			Elem: &schema.Resource{
+				Schema: map[string]*schema.Schema{
+					"bucket": {
+						Type:        schema.TypeString,
+						Required:    true,
+						Description: "The name of the bucket to grant the key access to.",
+					},
+					"permission": {
+						Type:        schema.TypeString,
+						Required:    true,
+						Description: "The permission to grant the key. Valid values are `read`, `readwrite`, or `fullaccess`.",
+					},
+				},
+			},
+		},
+		"access_key": {
+			Type:        schema.TypeString,
+			Description: "The access key for the Spaces key",
+			Computed:    true,
+		},
+		"secret_key": {
+			Type:        schema.TypeString,
+			Description: "The secret key for the Spaces key",
+			Computed:    true,
```

Comment:
```suggestion
			Computed:    true,
			Sensitive:   true,
```

Looking at this again with fresh eyes, I'm realizing we should probably set this as sensitive.